### PR TITLE
add(hk service to pacbio flow)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description
closes: https://github.com/orgs/Clinical-Genomics/projects/80/views/1?pane=issue&itemId=71880138
closes: https://github.com/Clinical-Genomics/add-new-tech/issues/40
### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/apps/housekeeper/hk.py [hunk 1]
```diff
@@ -570,19 +570,19 @@ def add_bundle_and_version_if_non_existent(self, bundle_name: str) -> None:
         else:
             LOG.debug(f"Bundle with name {bundle_name} already exists")
 
-    def store_fastq_path_in_housekeeper(
+    def create_bundle_add_file_with_tags(
```
Comment (ChrOertlin) [inline cg/apps/housekeeper/hk.py:573]: I generalised this function so it can be re-used

Comment (diitaz93) [inline cg/apps/housekeeper/hk.py:573]:
```suggestion
    def create_bundle_and_add_file_with_tags(
```

**Review by diitaz93 — COMMENTED** [review]
Is looking good 👏

**Review by diitaz93 — APPROVED** [review]
Looks great 👍

**Review by diitaz93 — APPROVED** [review]
Looks great 👍

**Pull request merged**